DataConnectors/AkamaiSecurityEvents/Connector_CEF_Akamai.json

```diff
@@ -0,0 +1,118 @@
+{
+    "id": "AkamaiSecurityEvents",
+    "title": "Akamai Security Events",
+    "publisher": "Akamai",
+    "descriptionMarkdown": "The Akamai Security Events data connector provides the capability to ingest [Akamai Security Events](https://www.akamai.com/us/en/products/security/) events into Azure Sentinel using syslog. Refer to [Akamai SIEM Integration documentation](https://developer.akamai.com/tools/integrations/siem) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **AkamaiSIEMEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/Akamai/AkamaiSIEMEvent.txt)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Akamai Events",
+            "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"Akamai\"\n| where DeviceProduct == \"akamai_siem\""
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Countries", 
+            "query": "AkamaiSIEMEvent\n | summarize count() by AkamaiSiemCountry\n | top 10 by count_"
```

Comment:
AkamaiSiemCountry is not the column returned in AkamaiSIEMEvent kusto function. Is the query referring to 'SrcGeoCountry'? Can you please correct this query?

Reply:
done